src/main/java/training/subsystems/Elbow/ElbowSubsystem.java

```diff
@@ -0,0 +1,76 @@
+package training.subsystems.Elbow;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.SparkPIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+import static training.subsystems.Elbow.Constants.*;
+
+public class ElbowSubsystem extends GBSubsystem {
```

Comment:
```suggestion
public class Elbow extends GBSubsystem {
```
No need type in name